### DISCUSSION TITLE
`working-directory` field cannot allow env var like "$HOME/Desktop"

### DISCUSSION THREAD

**Discussion by archsyscall** [ghostty-org/ghostty#3981] [Q&A] [answered]
I'm currently using macOS 15.1.1 with Ghostty 1.0.0. I encountered an issue with the working-directory setting in the configuration file.

I set working-directory = $HOME/Desktop, expecting it to correctly interpret the $HOME environment variable. However, the working directory is instead set to /.

Is this the expected behavior?

For my use case, it seems I need to specify an absolute path, such as /Users/foobar/Desktop, instead of using an environment variable like $HOME.

Could you clarify if this is intentional or a potential bug? If it's intentional, are there any plans to support environment variables in the configuration?

Thank you!

**Comment by JakeWharton** [comment, marked as answer]
Dupe of https://github.com/ghostty-org/ghostty/issues/3328
